ddtrace/contrib/requests/connection.py

```diff
@@ -81,5 +85,10 @@ def _wrap_request(func, instance, args, kwargs):
                     span.set_tag(http.STATUS_CODE, response.status_code)
                     # `span.error` must be an integer
                     span.error = int(500 <= response.status_code)
+                    # Storing response headers in the span.
+                    # Note that response.headers is not a dict, but an iterable
+                    # requests custom structure, that we convert to a dict
+                    response_headers = dict(getattr(response, 'headers', {}).items())
```

Comment:
Are you sure?

```python
>>> resp = requests.get('..')
>>> resp.headers.__class__
<class 'requests.structures.CaseInsensitiveDict'>
>>> dict(resp.headers)
{ ... }
```

Reply:
You are right! I do not know why I came out with that `items()` thing.